events/javaland/index.html.slim

```diff
@@ -0,0 +1,44 @@
+---
+layout: microsite-multi-page
+hero_class: event-javaland
+
+nav: 
+  - title: About JavaLand
+    url: #
```

Comment:
You need to set the URL to navigate to here. 

See this example: https://github.com/jboss-developer/www.jboss.org/blob/458e09eec2ac6eab3f8946f922db2d97cb2d86c4/vjbug/index.html.slim#L8